compiler/test/stdlib/float32.test.gr

```diff
@@ -6,11 +6,11 @@ from Pervasives use { (-) as numberSub }
 
 // Constants Test
 // smoke test:
-assert infinity > 100000000.0f
+assert Infinityf > 100000000.0f
```

Comment:
@ospencer Thoughts?

Reply:
I would prefer to keep the bindings... the syntax is not obvious, and having the binding gets us documentation in the library that is very searchable. If we want, we can mention the syntax in the documentation. We should, of course, remove the old implementations in favor of just `let infinity = Infinityf`.